sites/game-client/ui/main-ui.mjs

```diff
@@ -76,41 +74,49 @@ class MainUI {
    */
   setMainView(overlay) {}
 
+  updateNotification = () => {
```

Comment:
Not sure why you're using arrow notation here instead of the usual function/method declaration? If you've issues with `this` being unexpected, then you can bind this into this.updateNotification() before passing the callback.